app/controllers/queue_items_controller.rb

```diff
@@ -0,0 +1,37 @@
+class QueueItemsController < ApplicationController
+  before_filter :require_user
+ 
+  def index
+    @queue_items = current_user.queue_items#.order(order: :asc)
```

Comment:
Tiny nitpick, perhaps you could remove the comment if you no longer need the code :)